jira/jira-connector-1.0.1/src/main/resources/connector.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<connector>
+	<component name="jira" package="org.wso2.carbon.connectors">
```

Comment:
Please correct the package name as "org.wso2.carbon.connector" 
https://wso2.org/jira/browse/ESBCONNECT-43